app/components/Navbar/menuLinks.js

```diff
@@ -106,6 +106,10 @@ export const menuLinks = {
     href: 'https://v1.yearn.finance',
     title: 'Back to the old v1 UI',
   },
+  'woof?': {
+    href: 'https://stg.galaxy.eco/yearn',
```

Comment:
This should link to https://galaxy.eco/yearn, not to a staging site.
Good to go after 17:00 UTC.